SPCTR-3526 Fix: Form - Resolved an issue wherein form submission not working, when ReCaptcha version is selected and enable ReCaptcha is disabled. (#3633)
brainstormforce/ultimate-addons-for-gutenberg: README.md

## Changelog ##

### x.x.x ###

> * Fix: Form - Resolved an issue wherein form submission not working, when ReCaptcha version is selected and enable ReCaptcha is disabled.

Reviewer: ```suggestion
* Fix: Form - Resolved an issue wherein form submissions would not work as intended when a ReCaptcha version is used and ReCaptcha is disabled.
```
